### PR TITLE
Curator: fix issue with link focus state

### PR DESCRIPTION
**Pull request by madhusudhand**
#### Changes proposed in this Pull Request:

**Before:**
image

**After:**
image

And move post-excerpt link styles to `theme.json`

image

#### Related issue(s):

none.

### CODE REVIEW

**Comment by scruffian** [comment]
Theme.json now also supports the `focus` pseudo state, so it would be interesting to try that on this theme,

**Comment by madhusudhand** [comment]
Moved all the pseudo states to `theme.json` for this theme in a different [PR here](https://github.com/Automattic/themes/pull/6150) because the changes are not mergeable before GB `13.6`

File: curator/style.css [hunk 1]
```diff
@@ -211,19 +212,6 @@ a:not(
...
-.wp-block-post-excerpt .wp-block-post-excerpt__more-text a {
-	font-family: var(--wp--preset--font-family--heading-font);
-	font-size: 1.375rem;
-	font-weight: 900;
-	letter-spacing: -0.44px;
-	color: var(--wp--preset--color--foreground);
-	text-decoration: none;
-	text-transform: uppercase;
-}
-
```
Comment (MaggieCabrera) [inline curator/style.css:226]: lovely to see!

**Review by mikachan — APPROVED** [review]
This looks good to go for me until we're able to merge https://github.com/Automattic/themes/pull/6150.

**Pull request merged**